config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -164,8 +164,8 @@ periodics:
           - --env=CONTAINER_IMAGE=registry-sandbox.k8s.io/pause:3.1 #TODO(ameukam): revert when registry.k8s.io is ready
           - --extract=ci/latest
           - --gcp-node-image=gci
-          - --gcp-master-size=e2-standard-32
-          - --gcp-node-size=e2-standard-8
+          - --gcp-master-size=n1-standard-32
+          - --gcp-node-size=n1-standard-8
```

Comment:
Let's leave nodes with e2 - we don't specify min architecture for them, and e2 are a bit cheaper.

Reply:
ah, you mean `--gcp-node-size=e2-standard-8`, right ?